packages/table/src/spectrum-table.css

```diff
@@ -0,0 +1,2248 @@
+        var(--spectrum-alias-background-color-transparent)
+    ); /* .spectrum-Table--quiet .spectrum-Table-row */
+}
+.spectrum-Table--quiet .spectrum-Table-row:hover {
+    background-color: var(
+        --spectrum-table-m-regular-quiet-row-background-color-hover,
+        var(--spectrum-alias-highlight-hover)
+    ); /* .spectrum-Table--quiet .spectrum-Table-row:hover */
+}
+.spectrum-Table--quiet .spectrum-Table-row.focus-visible,
+.spectrum-Table--quiet .spectrum-Table-row.is-focused {
+    background-color: var(
+        --spectrum-table-m-regular-quiet-row-background-color-hover,
+        var(--spectrum-alias-highlight-hover)
+    ); /* .spectrum-Table--quiet .spectrum-Table-row.focus-ring,
+   * .spectrum-Table--quiet .spectrum-Table-row.is-focused */
+}
+.spectrum-Table--quiet .spectrum-Table-row.is-focused,
+.spectrum-Table--quiet .spectrum-Table-row:focus-visible {
+    background-color: var(
+        --spectrum-table-m-regular-quiet-row-background-color-hover,
+        var(--spectrum-alias-highlight-hover)
+    ); /* .spectrum-Table--quiet .spectrum-Table-row.focus-ring,
+   * .spectrum-Table--quiet .spectrum-Table-row.is-focused */
+}
+.spectrum-Table--quiet .spectrum-Table-row:active {
+    background-color: var(
+        --spectrum-table-m-regular-quiet-row-background-color-down,
+        var(--spectrum-alias-highlight-down)
+    ); /* .spectrum-Table--quiet .spectrum-Table-row:active */
+}
+.spectrum-Table--quiet .spectrum-Table-row.is-selected {
+    background-color: var(
+        --spectrum-table-m-regular-quiet-row-background-color-selected,
+        var(--spectrum-alias-highlight-selected)
+    ); /* .spectrum-Table--quiet .spectrum-Table-row.is-selected */
+}
+.spectrum-Table--quiet .spectrum-Table-row.is-selected:hover {
+    background-color: var(
+        --spectrum-table-m-regular-quiet-row-background-color-selected-hover,
+        var(--spectrum-alias-highlight-selected-hover)
+    ); /* .spectrum-Table--quiet .spectrum-Table-row.is-selected:hover */
+}
+.spectrum-Table--quiet .spectrum-Table-row.is-selected.focus-visible,
+.spectrum-Table--quiet .spectrum-Table-row.is-selected.is-focused {
+    background-color: var(
+        --spectrum-table-m-regular-quiet-row-background-color-selected-key-focus,
+        var(--spectrum-alias-highlight-selected-hover)
+    ); /* .spectrum-Table--quiet .spectrum-Table-row.is-selected.focus-ring,
+   * .spectrum-Table--quiet .spectrum-Table-row.is-selected.is-focused */
+}
+.spectrum-Table--quiet .spectrum-Table-row.is-selected.is-focused,
+.spectrum-Table--quiet .spectrum-Table-row.is-selected:focus-visible {
+    background-color: var(
+        --spectrum-table-m-regular-quiet-row-background-color-selected-key-focus,
+        var(--spectrum-alias-highlight-selected-hover)
+    ); /* .spectrum-Table--quiet .spectrum-Table-row.is-selected.focus-ring,
+   * .spectrum-Table--quiet .spectrum-Table-row.is-selected.is-focused */
+}
```

Comment:
Needs a revisit to make sure we're scoping out all the styles that aren't used herein.